gcloud-java-storage/src/test/java/com/google/gcloud/storage/ITStorageTest.java

```diff
@@ -86,7 +86,8 @@ public static void afterClass() throws ExecutionException, InterruptedException
 
   @Test(timeout = 5000)
   public void testListBuckets() throws InterruptedException {
-    Iterator<BucketInfo> bucketIterator = storage.list(Storage.BucketListOption.prefix(BUCKET),
+    Iterator<Bucket> bucketIterator =
```

Comment:
Ah, `list` is eventually consistent, which is why we don't already do this.  I'll update the tests to use `blob.delete()`